app/routes/book_routes.py

```diff
@@ -7,10 +7,14 @@
 @books_bp.post("")
 def create_book():
     request_body = request.get_json()
-    title = request_body["title"]
-    description = request_body["description"]
 
-    new_book = Book(title=title, description=description)
+    try:
+        new_book = Book.from_dict(request_body)
+        
+    except KeyError as error:
+        response = {"message": f"Invalid request: missing {error.args[0]}"}
```

Comment:
👍 I like that we're going to show handling the error here, and nice example of getting data out of the error.